src/main/scala/com/ironcorelabs/FailedResult.scala

```diff
@@ -0,0 +1,25 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+
+/**
+ * A failure to edit the access list of a document.
+ *
+ * @param users a list of users whose access was to be granted/revoked
+ * @param groups a list of groups whose access was to be granted/revoked
+ */
+case class FailedResult(
```

Comment:
This could also be replaced by `List[UserOrGroupId]`

Reply:
@clintfred Is there a reason not to just make this the simpler type?